lib/blocking-io.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function () {
+  if (process.stdout._handle && typeof process.stdout._handle.setBlocking === 'function') {
```

Comment:
You have to check and call setBlocking for stdout and stderr separately. Otherwise it leads to this problem:
```
$ cat stdio.js
if (process.stdout._handle && typeof process.stdout._handle.setBlocking === 'function') {
  process.stdout._handle.setBlocking(true)
  process.stderr._handle.setBlocking(true)
}
process.stdout.write('Hello stdout.\n');
process.stderr.write('Hello stderr.\n');
process.exit(1);
```
```
$ node -v
v6.0.0
```
```
$ node stdio.js
Hello stdout.
Hello stderr.
```
```
$ node stdio.js >/dev/null
Hello stderr.
```
```
$ node stdio.js 2>/dev/null

# no output
```
```
$ node stdio.js 2>eee
$ cat eee
/tmp/stdio.js:3
  process.stderr._handle.setBlocking(true)
                        ^

TypeError: Cannot read property 'setBlocking' of undefined
```
See the fix used in uglify-js: https://github.com/mishoo/UglifyJS2/pull/1061/files